packages/sdks/src/components/block/components/interactive-element.lite.tsx

```diff
@@ -51,6 +59,16 @@ export default function InteractiveElement(props: InteractiveElementProps) {
     },
   });
 
+  // Use onUpdate to track prop changes (Mitosis equivalent of useEffect/useTask)
+  onUpdate(() => {
+    useTarget({
+      qwik: () => {
+        state.forceRenderCount = state.forceRenderCount + 1;
+      },
+      default: () => {},
+    });
+  }, [props.wrapperProps, props.block?.component?.options]);
```

Comment:
oh that's a lot simpler than what I tried. This works fine and reacts to any deeply nested change in `component.options`?